src/legacy/integrations/contact-input.tsx

```diff
@@ -471,21 +487,7 @@ const ContactInput: FC<ContactInput> = ({
 				}
 				return chip;
 			}
-			return {
```

Comment:
Why this part is removed?
I remember that this part try to handle contact with invalid mail(select an item from the dropdown do not means that you have a valid mail inside)

Reply:
My bad, I removed it by mistake during the merge